kfs-core/src/main/java/edu/arizona/kfs/vnd/document/validation/impl/VendorRule.java

```diff
@@ -64,4 +68,43 @@ private boolean processSupplierDiversityValidation(MaintenanceDocument document)
     	
 		return valid;
 	}
+      
+    @Override
+    protected boolean validateParentVendorTaxNumber(VendorDetail vendorDetail) {
+    	String taxNumber = vendorDetail.getVendorHeader().getVendorTaxNumber();
+
+        // no parent vendor tax number validation needed if the tax number is blank
+        if (StringUtils.isBlank(taxNumber)) {
+            return true;
+        }
+
+        return super.validateParentVendorTaxNumber(vendorDetail);
+    }
+    
+    @Override
+    protected boolean validateTaxTypeAndTaxNumberBlankness(VendorDetail vendorDetail) {
+        boolean valid = true;
+        boolean isParent = vendorDetail.isVendorParentIndicator();
+        String vendorTaxTypeCode = vendorDetail.getVendorHeader().getVendorTaxTypeCode();
+        if (!StringUtils.isBlank(vendorDetail.getVendorHeader().getVendorTaxNumber()) && VendorConstants.NONE.equals(vendorTaxTypeCode)) {
+        	if (isParent) {
+                putFieldError(VendorPropertyConstants.VENDOR_TAX_TYPE_CODE, VendorKeyConstants.ERROR_VENDOR_TAX_TYPE_CANNOT_BE_BLANK);
+            }
+            valid &= false;
+        }
+        else if (StringUtils.isBlank(vendorDetail.getVendorHeader().getVendorTaxNumber()) && !VendorConstants.NONE.equals(vendorTaxTypeCode)) {
+        	if (isParent) {
+                putFieldError(VendorPropertyConstants.VENDOR_TAX_TYPE_CODE, VendorKeyConstants.ERROR_VENDOR_TAX_TYPE_CANNOT_BE_SET);
+            }
+            valid &= false;
```

Comment:
Discussed with Natalia and confirmed coded like this in delivered code. Agreed not worth another commit to fix this.